### PR TITLE
zohosprints-plugin permission file

### PR DESCRIPTION
**Pull request by zsprints**
[HOSTING-894](https://issues.jenkins-ci.org/browse/HOSTING-894#add-comment)
Plugin repo : [zohosprints](https://github.com/jenkinsci/zohosprints-plugin)
# Description

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Maintainer checklist (not for requesters!)

- [x] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [x] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [x] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by slide** [comment]
Please use the check boxes in the PR template to verify the required items are provided.

File: permissions/zohosprints-plugin.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "zohosprints-plugin"
+github: "https://github.com/jenkinsci/zohosprints-plugin"
+paths:
+  - "io/jenkins/plugins"
```
Comment (slide) [inline permissions/zohosprints-plugin.yml]:
```suggestion
  - "io/jenkins/plugins/zohosprints"
```

File: permissions/zohosprints-plugin.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "zohosprints-plugin"
+github: "https://github.com/jenkinsci/zohosprints-plugin"
```
Comment (slide) [inline permissions/zohosprints-plugin.yml]:
```suggestion
github: "jenkinsci/zohosprints-plugin"
```

**Comment by slide** [comment]
```
Exception in thread "main" java.lang.IllegalStateException: User name not known to Artifactory: zsprints
```

Please verify this is the username for the Jenkins infrastructure (Jira, Artifactory, etc) and NOT the github username.

**Comment by zsprints** [comment]
Hi @slide  I am not able to change the permission file. Is there any special permission needs to edit the file. Kindly assist

**Comment by zsprints** [comment]
@slide artefact json has been created in permission job. But i am unable to find plugin in artefact repo.
Please assist

**Comment by slide** [comment]
Please use the developers mailing list for further assistance.

File: permissions/plugin-zohosprints.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+name: "zohosprints-plugin"
```
Comment (daniel-beck) [inline permissions/plugin-zohosprints.yml]: Wrong. This should be `artifactId`.

Comment (zsprints) [reply permissions/plugin-zohosprints.yml]: @daniel-beck  zohosprints is an artifactId.

Comment (daniel-beck) [reply permissions/plugin-zohosprints.yml]:
```suggestion
name: "zohosprints"
```

Comment (zsprints) [reply permissions/plugin-zohosprints.yml]: @daniel-beck I'll do necessary changes

**Pull request merged**